packages/eslint-plugin-internal/src/rules/no-relative-paths-to-internal-packages.ts

```diff
@@ -0,0 +1,71 @@
+import path from 'path';
+
+import { createRule } from '../util';
+
+export const REPO_ROOT = path.resolve(__dirname, '../../../..');
```

Comment:
tried to import this from `/packages/repo-tools/src/paths.mts` but ts was upset about some module stuff that I didn't really understand.